grouper/fe/routes.py

```diff
@@ -71,6 +72,7 @@
     (r"/users", UsersView),
     (r"/users/public-keys", UsersPublicKey),
     (r"/users/tokens", UsersUserTokens),
+    (r"/users/service_create", ServiceAccountCreate),
```

Comment:
consider making this consistent with the other service account paths.

also, I like `/service` as a prefix a bit more than the inscrutable `/svc`